src/components/ThemeButton.tsx

```diff
@@ -104,8 +102,6 @@ const ThemeButton = forwardRef<Ref, ButtonProps>(function ThemeButton(
       size="md"
       className={`${basedStyles}  ${colorStyles} ${padding} ${disabledStyles} ${className}`}
       ref={ref}
-      aria-disabled={isDisabled}
-      ariaCurrent
```

Comment:
Hi @DonovanAndrews300, is there a reason for removing `ariaCurrent` instead of replacing it with `aria-current`?

Reply:
No actually I misunderstood what React-Aria was doing I added it in